### PR TITLE
Measure used disk storage

### LINKED ISSUE #6614: Calculate storage use
## Detailed Description
In order to implement storage quotas, we need to capture the storage use of datasets on-disk. I would suggest to calculate that on import and perhaps as a regular cron job. I think it would be useful to store the storage use on a mag-level. 
The aggregated storage use per organization needs to be exposed via an API so that the frontend can display it in the organization page and use it to enforce upload blocks.
Remote datasets and symlinked layers should not be counted towards the storage quota.

### PR DESCRIPTION
**Pull request by fm3**
webKnossos now measures the storage used by the datasets of the organizations.
New config options `datastore.reportUsedStorage.enabled` and `webKnossos.fetchUsedStorage.interval`.
Note that the organization is re-scanned regularly, but dataset upload + delete trigger a refresh of said dataset directly.
Datastores report that they have reportUsedStorage.enabled to webKnossos, which then stores that info in the postgres datastores table.
Load distribution strategy assumes that measuring a single organization isn’t that huge an operation (optimized for multi-orga setup).

### Steps to test:
- turn on storage reporting as described above
- start up, after initial delay, the organization api request should include used storage
- test with multiple organizations that the load distribution logic works

### Issues:
- fixes #6614 

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [x] Needs datastore update after deployment
- [x] Ready for review

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -142,6 +145,7 @@ datastore {
     enabled = true
     interval = 1 minute
   }
+  reportUsedStorage.enabled = false
```
[the remark this answers is not in the document]
Comment (fm3) [reply conf/application.conf:155]: The config parameter is the source of truth here. Each datastore can have this in their config. The value is sent to the wk side in the periodic datastore status report. wk then stores the value in postgres per datastore.

File: app/models/organization/OrganizationService.scala [hunk 2]
```diff
@@ -34,15 +34,17 @@ class OrganizationService @Inject()(organizationDAO: OrganizationDAO,
         "pricingPlan" -> organization.pricingPlan
       )
     } else Json.obj()
-    Fox.successful(
+    for {
+      usedStorageBytes <- organizationDAO.getUsedStorage(organization._id)
```
[the remark this answers is not in the document]
Comment (fm3) [reply app/models/organization/OrganizationService.scala:40]: No, this entire method will fail (return Fox.failure) if there getUsedStorage cannot return a value.
It should, however, always be able to. It reports zeroes if no storage reports have been received for the relevant organization.

**Review by jstriebel — APPROVED** [review]
Thanks for the clarifications, LGTM :+1:

**Comment by fm3** [comment]
@philippotto could you have a quick look at the latest commit https://github.com/scalableminds/webknossos/pull/6685/commits/cd80b770ef0e760a808858ed9f1d6de59093beba ? I adapted the front-end to use the new measured value

**Pull request merged**